README.md

```diff
@@ -62,13 +67,13 @@ optional arguments:
                         https://github.com/<owner>/<repository>
   --upstream-branch UPSTREAM_BRANCH
                         branch of the upstream repository
-  --skip-keys SKIP_KEYS
+  --skip-keys SKIP_KEYS [SKIP_KEYS ...]
                         packages to skip during regeneration
 ```
 
 ### Testing Gentoo Ebuilds
 ```
-$ superflore-check-ebuilds -h
+$ superflore-check-ebuilds --help
```

Comment:
Usage shows only `-h` => change `--help` back to it.

Reply:
Should I change it to -h in other script calls as well, because they all show just -h in short list of options and "-h, --help" bellow that. I've changed it to --help for consistence with other calls.

Reply:
Yes, please.